ibm/service/vpc/resource_ibm_is_instance_template.go

```diff
@@ -239,6 +242,14 @@ func ResourceIBMISInstanceTemplate() *schema.Resource {
 										ForceNew:    true,
 										Description: "The CRN of the [Key Protect Root Key](https://cloud.ibm.com/docs/key-protect?topic=key-protect-getting-started-tutorial) or [Hyper Protect Crypto Service Root Key](https://cloud.ibm.com/docs/hs-crypto?topic=hs-crypto-get-started) for this resource.",
 									},
+									isInstanceTemplateVolAttTags: {
+										Type:        schema.TypeSet,
+										Optional:    true,
+										ForceNew:    true,
+										Elem:        &schema.Schema{Type: schema.TypeString, ValidateFunc: validate.InvokeValidator("ibm_is_volume", "tags")},
```

Comment:
validate.InvokeValidator("ibm_is_volume", "tags") should be 
validate.InvokeValidator("ibm_is_instance_template", "tags")...the first argument should be resource name .
And add in the next func if this validator is not added..

Check in all resources